app-near/src/app/api/ngcform/email/route.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { TRPCError } from "@trpc/server";
+import { type NextRequest } from "next/server";
+import { handleCarbonFootprintEmail } from "~/server/carbon-footprint/handleCarbonFootprintEmail";
+import { getHTTPStatusCodeFromError } from "@trpc/server/http";
+import { ErrorCode } from "~/types/enums/error";
+
+const anonymize = (data: Record<string, any>) => ({ ...data, email: "" });
```

Comment:
:green_circle: je me dis que comme ça au moins lorsqu'on lit les logs on ne se dit pas il n'y a pas d'email transmis, mais ok il y avait un email, on a masqué son contenu.

```suggestion
const anonymize = (data: Record<string, any>) => ({ ...data, email: "***" });
```